posthog/api/dashboard.py

```diff
@@ -210,6 +194,44 @@ def create(self, validated_data: Dict, *args: Any, **kwargs: Any) -> Dashboard:
 
         return dashboard
 
+    def _duplicate_tiles(self, dashboard: Dashboard, existing_tile: DashboardTile) -> None:
+        if existing_tile.insight:
+            new_data = {
+                **InsightSerializer(existing_tile.insight, context=self.context).data,
+                "id": None,  # to create a new Insight
+                "last_refresh": now(),
+                "name": existing_tile.insight.display_name + " (Copy)",
```

Comment:
It's not ideal to cast a `derived_name` to a user-provided `name`, because a derived name is ephemeral – it's a placeholder that may and will change whenever the insight definition is changed. As a placeholder, it's italicized, so looks different from custom names too. This coercion breaks that barrier.
To maintain this differentiation, duplicated versions of insights shouldn't pretend to have a user-provided `name` if the original didn't have one.